civictechprojects/static/css/styles.scss

```diff
@@ -22,6 +23,7 @@
 @import 'partials/LandingController';
 @import 'partials/MainHeader';
 @import 'partials/MainFooter';
+@import 'partials/project_signup'; // has a modal opacity rule required for bootstrap-modals?
```

Comment:
I see that we use the react-bootstrap fade class to make this fade in but in my experience it doesn't actually fade in, it just appears. May be worth revisiting this if we extend our use of modals.